content/webinars/infrastructure-as-code-workshop-2020-04-09/index.md

```diff
@@ -7,6 +7,7 @@ meta_desc: "In this workshop, Luke Hoban shows you the easy way to start using I
 featured: false
 
 aliases: [ "/events/workshop-san-francisco-2020-04-09" ]
+redirect_to: "/webinars/getting-started-with-infrastructure-as-code/"
```

Comment:
This 404s in the Pulumify preview. Should it?

Reply:
Yeah, there was a typo in the directory name for the page. I fixed the typo, thanks for catching!